contrib/devtools/test-security-check.py

```diff
@@ -15,10 +15,10 @@
 def write_testcode(filename):
     with open(filename, 'w', encoding="utf8") as f:
         f.write('''
-    #include <stdio.h>
+    #include <cstdio>
     int main()
     {
-        printf("the quick brown fox jumps over the lazy god\\n");
+        std::printf("the quick brown fox jumps over the lazy god\n");
```

Comment:
```suggestion
        std::printf("the quick brown fox jumps over the lazy god\\n");
```

Otherwise, it fails:
```
test1.cpp:5:21: warning: missing terminating " character
    5 |         std::printf("the quick brown fox jumps over the lazy god
      |                     ^
test1.cpp:5:21: error: missing terminating " character
    5 |         std::printf("the quick brown fox jumps over the lazy god
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
test1.cpp:6:1: warning: missing terminating " character
    6 | ");
      | ^
test1.cpp:6:1: error: missing terminating " character
    6 | ");
      | ^~~
test1.cpp: In function 'int main()':
test1.cpp:7:9: error: expected primary-expression before 'return'
    7 |         return 0;
      |         ^~~~~~
```

Reply:
GRRRRR!!!